iOS Example Frame/Podfile.lock

```diff
@@ -1,13 +1,9 @@
 PODS:
-  - Checkout (3.5.3):
+  - Checkout (4.0.0):
     - CheckoutEventLoggerKit (= 1.2.0)
   - CheckoutEventLoggerKit (1.2.0)
-  - Frames (3.5.3):
-    - Checkout (= 3.5.3)
-    - CheckoutEventLoggerKit (= 1.2.0)
-    - PhoneNumberKit (= 3.3.3)
-  - Frames/Tests (3.5.3):
-    - Checkout (= 3.5.3)
+  - Frames (4.0.0):
```

Comment:
Question if this change was never merged how was i able to run the pod install earlier today on main branch and get latest 4.0.0 ?. Am I missing something. 
<img width="1072" alt="Screenshot 2022-10-20 at 12 43 33" src="https://user-images.githubusercontent.com/59569588/196939581-cc0f8a05-cf79-4759-bb1f-1752c41c7059.png">

Reply:
I noticed your are updating the iOS example frame so makes sense now.